integration-tests/scripts/gen-config.ts

```diff
@@ -0,0 +1,44 @@
+import * as Bun from "bun";
+import { unlink } from "node:fs/promises";
+
+interface CreateConfigOpts {
+  bundler: "rollup" | "vite" | "webpack";
+  format: string;
+  detectFormat: string;
+  version: string;
+  detectVersion: string;
+  file_format: "ts" | "cjs";
+}
+
+export async function createConfig({
+  bundler,
+  format,
+  detectFormat,
+  version,
+  detectVersion,
+  file_format,
+}: CreateConfigOpts) {
+  const bundlerDir = `fixtures/generate-bundle-stats/${bundler}`;
+  const baseConfigPath = `${bundlerDir}/${bundler}-base.config.${file_format}`;
+  const outFileName = `${bundler}-${version}-${format}.config.${file_format}`;
+  const outFilePath = `${bundlerDir}/${outFileName}`;
+
+  const file = Bun.file(baseConfigPath);
+
+  const outFile = Bun.file(outFilePath);
+  const outFileExists = await outFile.exists();
+
+  if (outFileExists) {
+    await unlink(outFilePath);
+  }
+
+  const upperCaseVersion = version.toUpperCase();
+  const upperCaseDetectVersion = detectVersion.toUpperCase();
+
+  let contents = await file.text();
+  contents = contents.replaceAll(detectFormat, format);
+  contents = contents.replaceAll(detectVersion, version);
+  contents = contents.replaceAll(upperCaseDetectVersion, upperCaseVersion);
+
+  await Bun.write(outFilePath, contents);
```

Comment:
nit: would be nice to get some error logging on these methods so easier to debug tests in the future.

Reply:
Do you mean adding in some try catch's ?

Reply:
yup exactly!

Reply:
alrighty perf done!